client/reducers/auth.js

```diff
@@ -0,0 +1,46 @@
+import {getToken} from '../utils/tokens'
+
+const initialState = {
+  error: null,
+  loggedIn: !!getToken(), // Coerces object to boolean, so either returns true or false
+  newRegistration: false,
+  pending: false
+}
+
+export default function authReducer (state = initialState, action) {
+  switch (action.type) {
+    case 'SIGNIN_PENDING':
```

Comment:
what does this return?

Reply:
We can use multiple cases one after the other  which use the same logic.
case 'SIGNIN_PENDING':
case 'REGISTER_PENDING':
      return ...
would be logically the same as 
case 'SIGNIN_PENDING' || 'REGISTER_PENDING':
      return ...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch